data/noiseprofiles.json

```diff
@@ -850,6 +850,21 @@
             {"skip": true, "name": "EOS 200D iso 51200", "iso": 51200, "a": [0.00249000518027299, 0.00112357559710744, 0.00181437909489014], "b": [1.72431912704092e-05, 2.76503246829995e-05, 2.03920295934623e-05]}
           ]
         },
+        {
+          "comment": "EOS 200D II (EOS 250D) contributed by scybernaut",
+          "model": "EOS 250D",
+          "profiles": [
+            {"name": "EOS 200D II iso 100", "iso": 100, "a": [5.22367254613872e-06, 2.38125889064614e-06, 4.62535308187296e-06], "b": [4.05699863614146e-09, 2.99296859087643e-09, 4.09607381684299e-09]},
```

Comment:
Change all "EOS 200D II" occurrences to "EOS 250D" please.

Reply:
done